x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/ApiKeyIntegTests.java

```diff
@@ -2488,6 +2739,7 @@ private void verifyGetResponse(
         List<CreateApiKeyResponse> responses,
         List<Map<String, Object>> metadatas,
         List<RoleDescriptor> expectedRoleDescriptors,
+        List<RoleDescriptor> expectedLimitedRoleDescriptors,
```

Comment:
Most changes in this file are due to this method signature change which allows checking the limited-by role descriptors.